### ISSUE TITLE
Changing speech style switches the language

### ISSUE THREAD

**Issue by NSoiffer** [NSoiffer/MathCAT#262]
From Määttä Sami:
> I have just now updated my own branch. Now I’m using NVDA 2024.1 and the linked MathCAT version. I noticed a bug with this configuration:

> Changing the speech mode between ClearSpeak and SimpleSpeak defaults the speech language to English without indication in the menu. As in the chosen language doesn’t change, it says it’s Finnish even though the synth outputs English. Changing the language in the menu to something else and then back to Finnish fixes this or re-opening NVDA.

> I have the older version of NVDA (2023.x) and MathCAT on my work laptop and it doesn’t have that bug.

I tried to replicate it in MathCAT proper, but couldn't. I can replicate it (with Spanish) from the dialog.

**Comment by NSoiffer** [comment]
In MathCAT, I tried changing the fallback default and the language changed. That's a pretty clear sign that the problem does actually reside in MathCAT proper.
